tests/types/src/valid.ts

```diff
@@ -435,6 +440,11 @@ const retrievedLinkAuthenticationElement: StripeLinkAuthenticationElement | null
   'linkAuthentication'
 );
 
+let shippingAddressElementDefaults: StripeShippingAddressElement = elements.create(
+  'shippingAddress'
+);
+shippingAddressElementDefaults = elements.create('shippingAddress');
```

Comment:
Just curious, why create the element twice?

Reply:
Ah, good catch! The second time was supposed to be with empty options `{}`

Reply:
Fixed now!